### PR TITLE
Pylint edits and fixing import errors

### PR DESCRIPTION
**Pull request by stevehenny**

### CODE REVIEW

**Comment by jgoeders** [comment]
@stevehenny It looks like this just needs formatting run before it is good to go.  

You can run `make format` (https://github.com/byuccl/yinstruments/blob/37d454eb9e68798b5a72fe25e3b6ef2ea8e0605a/Makefile#L15) to fix formatting.  

Also, if you set up your VS Code settings correctly, it will just auto-format for you each time you save a Python file.  For example, I have this in my VS Code user settings:

```json
    "[python]": {
        "editor.formatOnSave": true,
        "editor.defaultFormatter": "ms-python.black-formatter",
        "editor.formatOnType": true,
    },
    "python.formatting.provider": "black",
    "python.formatting.blackArgs": [
        "-l",
        "100"
    ],

```

**Comment by stevehenny** [comment]
Okay I'll go ahead and format it. I also just remembered that Garret wanted there to be an is_on function for both the Lindy and the Netbooter so I'll add that and then push things up so we can merge.

File: yinstruments/.vscode/settings.json
Comment (jgoeders) [inline yinstruments/.vscode/settings.json:1]: Can you remove this file?  We only want .vscode files at the top level directory.

File: yinstruments/pdu/README.md [hunk 1]
```diff
@@ -6,9 +6,19 @@ see several classes that help organization and drive connections to different
 PDUs. The PDU class is the parent class to the Netbooter and Lindy. All of these 
 classes have the following callable functions: get_status, reboot, off, and on.
```
Comment (jgoeders) [inline yinstruments/pdu/README.md:8]: Can you add two subsections here:

## Package
  * briefly mention/show how to import and instance your the pdu classes in the package

## CLI
  * Here you can keep the text below that talks about *SETUP:* and python environments.

File: yinstruments/pdu/README.md [hunk 2]
```diff
@@ -6,9 +6,19 @@ see several classes that help organization and drive connections to different
...
 
+SETUP: There are two things that you will need to do before the commands provided will work:
+
+    1. make python enviornment and go into enviornment
+        --> make env
+        --> source .venv/bin/activate
```
Comment (jgoeders) [inline yinstruments/pdu/README.md]: These only apply to the CLI and can remain in that section.

File: yinstruments/pdu/README.md [hunk 3]
```diff
@@ -6,9 +6,19 @@ see several classes that help organization and drive connections to different
...
+
+    2. install snmp package into your enviornment
+        --> sudo apt update
+        --> sudo apt install snmp snmpd
```
Comment (jgoeders) [inline yinstruments/pdu/README.md]: This applies for CLI or packages usage, so you might want to move it before the package/CLI subsections.

File: yinstruments/pdu/README.md [hunk 4]
```diff
@@ -6,9 +6,19 @@ see several classes that help organization and drive connections to different
...
+
 
 HOW TO RUN: You will need 4 command line arguments NOT including this python file. 
 The command line call will look something like this:
         
-    python3 cli.py <dev_type> <ip_address> <command> <port_num>
+    pdu <dev_type> <ip_address> <command> <port_num>
```
Comment (jgoeders) [inline yinstruments/pdu/README.md]: Can you provide an example?  And mention the possible *dev_type* values?

**Review by jgoeders — CHANGES_REQUESTED** [review]

**Review by jgoeders — APPROVED** [review]

**Pull request merged**